tools/test-proxy/Azure.Sdk.Tools.TestProxy/Admin.cs

```diff
@@ -101,26 +102,25 @@ public async void SetMatcher()
 
         public object GetSanitizer(string name, JsonDocument body)
         {
-            return GenerateInstance("Azure.Sdk.Tools.TestProxy.Sanitizers.", name, body);
+            return GenerateInstance("Azure.Sdk.Tools.TestProxy.Sanitizers.", name, new string[] { "value" }, documentBody: body);
         }
 
         public object GetTransform(string name, JsonDocument body)
         {
-            return GenerateInstance("Azure.Sdk.Tools.TestProxy.Transforms.", name, body);
+            return GenerateInstance("Azure.Sdk.Tools.TestProxy.Transforms.", name, new string[]{}, documentBody: body);
         }
 
         public object GetMatcher(string name, JsonDocument body)
         {
-            return GenerateInstance("Azure.Sdk.Tools.TestProxy.Matchers.", name, body);
+            return GenerateInstance("Azure.Sdk.Tools.TestProxy.Matchers.", name, new string[]{}, documentBody:body);
         }
 
-        public object GenerateInstance(string typePrefix, string name, JsonDocument body = null)
+        private object GenerateInstance(string typePrefix, string name, string[] acceptableEmptyArgs, JsonDocument documentBody = null)
```

Comment:
nit: is it worth using a HashSet for acceptableEmptyArgs to avoid the O(N) lookup?

Reply:
I might as well, it's a small ask 👍